drivers/SmartThings/zwave-sensor/src/test/test_fibaro_motion_sensor.lua

```diff
@@ -163,11 +163,50 @@ test.register_message_test(
     {
       channel = "capability",
       direction = "send",
-      message = mock_device:generate_test_message("main", capabilities.motionSensor.motion.active())
+      message = mock_device:generate_test_message("main", capabilities.accelerationSensor.acceleration.active())
```

Comment:
why's this test changing?

Reply:
Because I've added the subdriver which handles SensorAlarm in the same way as here [devicetypes/smartthings/fibaro-motion-sensor.src/fibaro-motion-sensor.groovy#L215](https://github.com/SmartThingsCommunity/SmartThingsPublic/blob/78747fa5a3b42a0c801f1095f8f7521da968ba42/devicetypes/smartthings/fibaro-motion-sensor.src/fibaro-motion-sensor.groovy#L215)